src/pages/index.js

```diff
@@ -5,28 +5,29 @@
  * SPDX-FileCopyrightText: 2021 Siemens AG
  */
 
-import React from 'react';
-import clsx from 'clsx';
-import Layout from '@theme/Layout';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import styles from './index.module.css';
-import ParticipantImages from '../components/ParticipantImages';
+import React from "react";
+import clsx from "clsx";
+import Layout from "@theme/Layout";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import styles from "./index.module.css";
+import ParticipantImages from "../components/ParticipantImages";
 
 function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
+  const { siteConfig } = useDocusaurusContext();
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
+    <header className={clsx("hero hero--primary", styles.heroBanner)}>
       <div className="container">
         <h1 className="hero__title">{siteConfig.title}</h1>
         <p className="hero__subtitle">{siteConfig.tagline}</p>
         <p className="hero__subtitle">
           Navigate the projects from the drop-downs in the top.
         </p>
         <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/2021">
+          <Link className="button button--secondary button--lg" to="/docs/2022">
```

Comment:
Buttons appear a little wired here. Can you please add a space between them?
![image](https://user-images.githubusercontent.com/18077542/153591702-a0ebf05a-9f96-4536-b768-a825c06b61b6.png)